backend/api/src/get-singup-bonus.ts

```diff
@@ -0,0 +1,25 @@
+import { APIError, APIHandler } from 'api/helpers/endpoint'
+import { getUser } from 'shared/utils'
+import { MARKET_VISIT_BONUS_TOTAL } from 'common/economy'
+import { sendOnboardingMarketVisitBonus } from 'shared/onboarding-helpers'
+import * as admin from 'firebase-admin'
+
+export const getSingupBonus: APIHandler<'get-signup-bonus'> = async (
```

Comment:
Typo, plus I don't think it should start with 'get' if we are going to be using endpoints for just fetching data. Maybe 'claim-signup-bonus'?